MergerLogic/Utils/httpUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MergerLogic.Batching;
+
+namespace MergerLogic.Utils
+{
+    internal class httpUtils : DataUtils
+    {
+        private PathPatternUtils _pathPatternUtils;
```

Comment:
Convention is using `mPropertyName`.

Reply:
???

Reply:
use this conventions
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions